resources/lang/pl/users.php

```diff
@@ -43,18 +43,22 @@
         ],
     ],
 
+    'card' => [
+        'loading' => 'Ładowanie...',
+    ],
+
     'login' => [
         '_' => 'Zaloguj się',
-        'locked_ip' => 'Twój adres IP jest zablokowany. Poczekaj kilka minut.',
+        'locked_ip' => 'Twój adres IP został zablokowany. Poczekaj kilka minut.',
         'username' => 'Nazwa użytkownika',
         'password' => 'Hasło',
         'button' => 'Zaloguj się',
         'button_posting' => 'Logowanie...',
         'remember' => 'Zapamiętaj ten komputer',
         'title' => 'Zaloguj się, aby kontynuować',
         'failed' => 'Niepoprawny login/hasło',
-        'register' => 'Nie posiadasz konta osu! ? Stwórz nowe',
-        'forgot' => 'Zapomniałeś hasła?',
+        'register' => 'Nie posiadasz konta osu!? Utwórz nowe',
+        'forgot' => 'Nie pamiętasz hasła?',
         'beta' => [
             'main' => 'Dostęp do bety jest obecnie ograniczony do wybranych użytkowników.',
             'small' => '(supporterzy dostaną go wkrótce)',
```

Comment:
to w końcu w polskim tłumaczeniu ma być *supporter* czy *donator*?

also *wkrótce otrzymają go supporterzy*

Reply:
A no ta donator